### PR TITLE
ROOT proposal

### PR DESCRIPTION
**Pull request by couet**
ROOT documentation project proposal

### CODE REVIEW

File: _gsdocs-proposals/2020/proposal_ROOTdocumentation.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+- Technical specifications for the new data format RNTuple.
+
+## Project duration
+
+This is expected to be a long (6 months) project which will possibly continue
+after this period.
+
+## Related material
+
+  1. https://root.cern
```
Comment (aflp91) [inline _gsdocs-proposals/2020/proposal_ROOTdocumentation.md:45]: List items should be linkable ([Root website](https://root.cern))

Comment (couet) [reply _gsdocs-proposals/2020/proposal_ROOTdocumentation.md]: Done

File: _gsdocs-proposals/2020/proposal_ROOTdocumentation.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  4. https://root.cern/doc/master/
+  5. https://root.cern/doc/master/group__Tutorials.html
+  6. https://github.com/root-project/web
+  7. https://root-project.github.io/web/manual/
+
+## Expected results
+
+The deliverables of the projects will be directly visible online
+in the various area mentioned in the "Tasks" paragraph. The writers
+will directly commit his work as pull requests to the web site repository.
```
Comment (agheata) [inline _gsdocs-proposals/2020/proposal_ROOTdocumentation.md:57]: In the tasks section it is expected to have the description of the tasks, while here what are the minimum results of these tasks to consider the project successful. Please expand along this line.

File: _gsdocs-proposals/2020/proposal_ROOTdocumentation.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+## Experience required
+
+We are seeking technical writers with a good experience.
+Technical writing skills are mandatory.
+The following technical skills are mandatory:
```
Comment (agheata) [inline _gsdocs-proposals/2020/proposal_ROOTdocumentation.md:63]: Besides good technical writer skills, the following additional technical skills are mandatory:

**Pull request merged**